<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SkyViper GroundStation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Icons/Icons/dish8.png</normaloff>:/Icons/Icons/dish8.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>48</width>
    <height>48</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="toolBarMain">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="toolbarLogs"/>
   <addaction name="toolbarGeolocation"/>
   <addaction name="toolbarTelemetry"/>
   <addaction name="toolbarFPV"/>
   <addaction name="toolbarSystemInfo"/>
   <addaction name="toolbarAlerts"/>
   <addaction name="toolbarSteering"/>
   <addaction name="toolbarEmulator"/>
  </widget>
  <action name="menuFileClose">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/power27.png</normaloff>:/Icons/Icons/power27.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="menuOptionsCommunication">
   <property name="text">
    <string>Communication</string>
   </property>
  </action>
  <action name="menuHelpQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="menuHelpGroundStation">
   <property name="text">
    <string>About ARMCopter GroundStation</string>
   </property>
  </action>
  <action name="toolbarLogs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/files12.png</normaloff>:/Icons/Icons/files12.png</iconset>
   </property>
   <property name="text">
    <string>Logs</string>
   </property>
   <property name="iconText">
    <string>     Logs     </string>
   </property>
   <property name="toolTip">
    <string>Logs</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="toolbarGeolocation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/map30.png</normaloff>:/Icons/Icons/map30.png</iconset>
   </property>
   <property name="text">
    <string>Geolocation</string>
   </property>
   <property name="toolTip">
    <string>Map &amp; Geolocation</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="toolbarTelemetry">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/chart47.png</normaloff>:/Icons/Icons/chart47.png</iconset>
   </property>
   <property name="text">
    <string>Telemetry</string>
   </property>
   <property name="iconText">
    <string> Telemetry </string>
   </property>
   <property name="toolTip">
    <string>Telemetry &amp; Statistics</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="toolbarFPV">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/video71.png</normaloff>:/Icons/Icons/video71.png</iconset>
   </property>
   <property name="text">
    <string>FPV</string>
   </property>
   <property name="iconText">
    <string>     FPV      </string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="toolbarSystemInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/computer32.png</normaloff>:/Icons/Icons/computer32.png</iconset>
   </property>
   <property name="text">
    <string>System Info</string>
   </property>
   <property name="iconText">
    <string>System Info</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="toolbarAlerts">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/alarm57.png</normaloff>:/Icons/Icons/alarm57.png</iconset>
   </property>
   <property name="text">
    <string>Alerts</string>
   </property>
   <property name="iconText">
    <string>    Alerts    </string>
   </property>
   <property name="toolTip">
    <string>Alerts &amp; Notifications</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="toolbarSteering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/videogame10.png</normaloff>:/Icons/Icons/videogame10.png</iconset>
   </property>
   <property name="text">
    <string>Steering</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="toolbarEmulator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/screen213.png</normaloff>:/Icons/Icons/screen213.png</iconset>
   </property>
   <property name="text">
    <string>Emulator</string>
   </property>
   <property name="toolTip">
    <string>Emulator</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
